x-pack/plugins/stack_connectors/server/connector_types/cases/jira/service.ts

```diff
@@ -136,6 +136,9 @@ export const createExternalService = (
     }, '');
   };
 
+  const escapeSpecialCharacters = (str: string) =>
```

Comment:
nit: `escapeJqlSpecialCharacters`

Reply:
fixed in [a73197](https://github.com/elastic/kibana/pull/145610/commits/a73197ef9e85657180c33cd6cee4290f9b8e655a)